### PR TITLE
Types update for TypeScript developers.

### PR DESCRIPTION
**Pull request by AhmedMKamal**
Updated widthPercentageToDP and heightPercentageToDP methods to accept numeric values.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -15,7 +15,9 @@ let screenHeight = Dimensions.get('window').height;
  */
 const widthPercentageToDP = widthPercent => {
   // Parse string percentage input and convert it to number.
-  const elemWidth = parseFloat(widthPercent);
+  const elemWidth = parseFloat(
+    typeof widthPercent === "number" ? widthPercent : widthPercent.toString()
```
Comment (marudy) [inline index.js]: why not do it 
```
const elemWidth = typeof widthPercent === "number" ? parseFloat(widthPercent)
```

**Comment by marudy** [comment]
Hey @AhmedMKamal thanks for the PR. I made a comment over your solution. If you agree, we can change it, then merge and publish a new version. Cheers man

File: index.js (hunk 1 above)
Comment (AhmedMKamal) [reply index.js]: You mean `const elemWidth = typeof widthPercent === "number" ? widthPercent : parseFloat(widthPercent)`?

**Comment by AhmedMKamal** [comment]
Hi, @marudy I pushed the update as you mentioned.

**Comment by marudy** [comment]
Thanks @AhmedMKamal. Did you test it as well for both scenarios ? Input as string and input as number ? If so I will issue the release today.

Thanks again

**Comment by AhmedMKamal** [comment]
Hi @marudy, Take a look at [this snack](https://snack.expo.io/@ahmedmkamal/react-native-responsive-screen).

**Comment by AhmedMKamal** [comment]
@marudy Cheers

**Pull request merged**